src/index.html

```diff
@@ -11,7 +11,215 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+
+      <fieldset
+        class="block"
+      >
```

Comment:
This code style require only in cases when tag have 3 and more attributes. or line length takes more then 80